recipes/pybobyqa/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pybobyqa" %}
+{% set version = "1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/numericalalgorithmsgroup/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 16d8a86e7f9efcf38c6b3ad19d64091dbb201ac1a8d06ad66e28489e5cb3aa97
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - numpy >=1.11
+    - scipy >=0.17
+    - pandas >=0.17
+
+test:
+  imports:
+    - pybobyqa
+
+about:
+  home: https://github.com/numericalalgorithmsgroup/pybobyqa
+  summary: 'Derivative-Free Solver for Bound-Constrained Minimization'
+  description: |
+    Py-BOBYQA is a flexible package for solving bound-constrained general 
+    objective minimization, without requiring derivatives of the objective. At 
+    its core, it is a Python implementation of the BOBYQA algorithm by Powell, 
+    but Py-BOBYQA has extra features improving its performance on some problems
+    (see the papers below for details). Py-BOBYQA is particularly useful when 
+    evaluations of the objective function are expensive and/or noisy.
+  license: GPL-3.0-or-later
+  license_file: LICENSE.txt
+  doc_url: https://numericalalgorithmsgroup.github.io/pybobyqa/
+
+extra:
+  recipe-maintainers:
+    - vitenti
+    - marcpaterno
```

Comment:
All maintainers on the recipe besides the person who submitted it need to comment here to confirm they are ok maintaining the recipe.

Reply:
I am OK maintaining this recipe.